### ISSUE TITLE
Selected area in grab-canvas is not correct

### ISSUE THREAD

**Issue by TimPietrusky** [vcync/modV#254]
## Issue Description
When using grab-canvas and a selection that can not be divided into an Integer, but a Float, the resulting colors array gives weird results. 

All of this happens because the [result of the following calculation](https://github.com/m-o-d-v/modV/blob/2.0-drawloop/src/extra/grab-canvas/luminave-connector.js#L214) is a float and it should be an Integer: 

```js
    // Size of each area
    const areaSize = (this.width / areaAmount) + (this.height / areaAmount);
```

## Proposed Solution
```js
    // Size of each area
    const areaSize = Math.floor((this.width / areaAmount) + (this.height / areaAmount));
```

**Comment by TimPietrusky** [comment]
@2xAA Where should this land? Which branch is considered "stable" and can be used for performances?

**Comment by 2xAA** [comment]
Okay - you're in charge of GrabCanvas really haha, so if that'll fix it please make a PR.

At the moment, all work has been on 2.0-drawloop.
This really needs to be merged into 2.0.

I can do this, then please create the PR to 2.0.

**Comment by 2xAA** [comment]
Merged in #255

**Comment by TimPietrusky** [comment]
This is still broken when using x: 11 and y: 6. 

The quick fix was to `Math.floor` the parameters in `getAverage()`, but has to be investigated further.
